README.md

```diff
@@ -95,12 +95,22 @@ intended for production deployments. Baby Buddy is deployed to Docker Hub as
 [babybuddy/babybuddy](https://hub.docker.com/r/babybuddy/babybuddy) so this is
 the only file needed for a Docker deployment with Docker Compose.
 
-1. Copy the contents of `docker-compose.example.yml` as `docker-compose.yml`
-and set, at least, the `ALLOWED_HOSTS` and `SECRET_KEY` variables under
+A secondary example file `docker-compose.example.sqlite.yml` is also available
+for a simpler SQLite-based deployment (the default example users PostgreSQL).
+
+1. Copy the raw content of either `docker-compose.example.yml` or `docker-compose.example.sqlite.yml` 
+into a new file named `docker-compose.yml` 
+
+        wget https://raw.githubusercontent.com/babybuddy/babybuddy/master/docker-compose.example.yml
```

Comment:
Lets make this this `wget -O docker-compose.yml [...]`. That way the file downloads with the correct name.